desktop-widgets/mainwindow.ui

```diff
@@ -453,7 +453,7 @@
     <bool>true</bool>
    </property>
    <property name="icon">
-    <iconset resource="../subsurface.qrc">
+    <iconset>
```

Comment:
And this REALLY worries me.
TBH, I'd much prefer if you edited the diff to just  contain the removal of the actionDownloadWeb and not touch anything else. I know that's paranoid, but...